test/t8n/t8n.cpp

```diff
@@ -121,11 +135,14 @@ int main(int argc, const char* argv[])
 
                         j_receipt["blockHash"] = NULL_HEXSTRING_32;
                         j_receipt["contractAddress"] = NULL_HEXSTRING_20;
-                        j_receipt["logsBloom"] = NULL_HEXSTRING_256;
+                        j_receipt["logsBloom"] = hex0x(receipt.logs_bloom);
                         j_receipt["logs"] = json::json::array();  // FIXME: Add to_json<state:Log>
                         j_receipt["root"] = "";
                         j_receipt["status"] = "0x1";
                         j_receipt["transactionIndex"] = hex0x(idx);
+                        // TODO: Make sure that tx is not copied here.
```

Comment:
Hm? The move ensures it is not copied?

Reply:
This comment was from the previous version w/o move